jetty-client/src/main/java/org/eclipse/jetty/client/MultiplexConnectionPool.java

```diff
@@ -80,19 +82,13 @@ private static int ceilDiv(int a, int b)
     @Override
     public int getMaxMultiplex()
     {
-        synchronized (this)
-        {
-            return maxMultiplex;
-        }
+        return lock.runLocked(() -> maxMultiplex);
     }
 
     @Override
     public void setMaxMultiplex(int maxMultiplex)
     {
-        synchronized (this)
-        {
-            this.maxMultiplex = maxMultiplex;
-        }
+        lock.runLocked(() -> this.maxMultiplex = maxMultiplex);
     }
```

Comment:
Would this be a better pattern to avoid allocation?
```suggestion
   private void setMaxMultiplexLocked(int maxMultiplex)
   {
       this.maxMultiplex = maxMultiplex
   }
   @Override
   public void setMaxMultiplex(int maxMultiplex)
   {
       lock.runLocked(this::setMaxMultiplexLocked);
   }
```

Reply:
Uh, no. I won't write another method just for this, if you don't want to allocate ever, just use the `try` version.